client/gutenberg/editor/browser-url/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { withSelect } from '@wordpress/data';
+
+/**
+ * After making changes to a new post, this component will update the url to append the post id:
+ *
+ * For example:
+ * https://wordpress.com/gutenberg/post/mysiteslug
+ *
+ * https://wordpress.com/gutenberg/post/mysiteslug/1234
+ */
+export class BrowserURL extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			historyId: null,
+		};
+	}
```

Comment:
Is this format to keep the file as similar as possible to the Guten version?
Couldn't we move the state as a class property?

Reply:
Copy pasta removed in 41d9a0e